internal/app/machined/internal/server/v1alpha1/v1alpha1_server.go

```diff
@@ -1754,7 +1755,7 @@ func (s *Server) EtcdRemoveMember(ctx context.Context, in *machine.EtcdRemoveMem
 
 	ctx = clientv3.WithRequireLeader(ctx)
 
-	if err = client.RemoveMember(ctx, in.Member); err != nil {
```

Comment:
I'm almost positive we should have separate (new) API which removes by member ID, and use it in proper way, while keeping old API deprecated.